framework/wazuh/rbac/orm.py

```diff
@@ -1272,6 +1272,9 @@ class PoliciesManager:
     This class is the manager of the Policies, this class provided
     all the methods needed for the policies administration.
     """
+    def __init__(self):
+        self.action_regex = r'^[a-zA-Z_\-]+:[a-zA-Z_\-]+$'
+        self.resource_regex = r'^[a-zA-Z_\-*]+:[\w_\-*]+:[\w_\-\/.*]+$'
```

Comment:
Wouldn't it be more appropriate if these 2 instance variables were class variables? Their values will always be the same in all instances of the class.